website/content/docs/get-started/operations-qs.mdx

```diff
@@ -0,0 +1,190 @@
+   $ vault write auth/userpass/users/opsuser \
+       password=p@ssw0rd
+   ```
+
+   The user `opsuser` is created. The password is `p@ssw0rd`.
+
+1. (Optional) Enable the `approle` auth method. This is a general purpose plugin that
+   allows machines or apps to authenticate to Vault.
+
+   ```shell-session
+   $ vault auth enable approle
+   ```
+
+   The `approle` auth method is enabled.
+
+1. (Optional) To authenticate workloads with Vault, you must create a role.
+
+   ```shell-session
+   $ vault write auth/approle/role/my-app-role \
+       secret_id_ttl=10m \
+       token_ttl=20m \
+       token_max_ttl=30m
+   ```
+
+   The role `my-app-role` is created. The role has a secret ID TTL of 10 minutes, a
+   token TTL of 20 minutes, and a token max TTL of 30 minutes.
+
+## Enable secrets engines
+
+Vault supports multiple [secrets engines](/vault/docs/secrets). These
+plugins support various use cases like storing static secrets, dynamic secrets,
+and managing PKI certificates.
+
+Enable the `kv` secrets engine. This engine stores arbitrary secrets.
+
+```shell-session
+$ vault secrets enable -version=2 kv
+```
+
+The `kv` secrets engine is enabled.
+
+## Store a secret
+
+Vault securely stores secrets static for a configured `kv` secrets engine.
+
+Write a secret to the `kv` secrets engine.
+
+```shell-session
+$ vault kv put kv/creds username=opsuser password=p@ssw0rd
+```
+
+The `kv put` command writes the secret to the `kv` secrets engine. In this example, two
+secrets are written: The `username` key with a value of `opsuser` and the key
+`password` with a value of `p@ssw0rd`.
+
+## Protect secrets
+
+In the previous step, you wrote a password to the `kv` secrets engine using the
+root token. Using the root token, and root policy, is not recommended for production environments.
+
+Vault policies allow you to control access to secrets, and which operations are allowed.
+
+1. Write a policy that permits access to the `kv` secrets engine.
+
+   ```shell-session
+   $ vault policy write kv-access-policy - << EOF
+   path "kv/creds" {
+      capabilities = ["read", "create", "update", "delete"]
+   }
+   EOF
+   ```
+
+   This policy supports the `read`, `create`, `update`, and `delete` operations on
+   the path `kv/creds`.
+
+1. Attach the policy to the `opsuser` user.
+
+   ```shell-session
+   $ vault write auth/userpass/users/opsuser \
+       policies=kv-access-policy
+   ```
+
+   The `opsuser` user is updated with the `kv-access-policy` policy.
+
+## Authenticate to Vault
+
+Authenticate to Vault using the `opsuser` user. When prompted, enter the
+password `p@ssw0rd`.
+
+```shell-session
+$ vault login -method=userpass username=opsuser
+```
+
+You are now authenticated to Vault.
+
+## Retrieve a secret
+
+You authenticated to Vault as the `opsuser` user. Retrieve the secrets from the
+path `kv/creds`. The `opsuser` user has access to the `kv/creds` path from the
+`kv-access-policy`.
+
+```shell-session
+$ vault kv get kv/creds
+```
```

Comment:
````suggestion

Try retrieving the secrets stored on the `kv/creds` path:

```shell-session
$ vault kv get kv/creds
```

Now try retrieving the secrets stored on the `kv/api-keys` path:

```shell-session
$ vault kv get kv/api-keys
```

The second command fails because the policy for `opsuser` does
not grant access to the `/api-keys` path.
````